redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -301,14 +301,13 @@ internal class LazyListContainerCell(
     }
 
     // Confirm the cell is bound when it's about to be displayed.
-    if (superview == null && newSuperview != null) {
-      require(binding!!.isBound) { "about to display a cell that isn't bound!" }
+    if (superview == null && newSuperview != null && !binding!!.isBound) {
+      binding!!.bind(this)
     }
 
     // Unbind the cell when its view is detached from the table.
     if (superview != null && newSuperview == null) {
       binding?.unbind()
-      binding = null
```

Comment:
So we can bind again later

Reply:
This feels a little dangerous to me--what are the odds that this causes a memory management issue, because it's being retained?